README.md

```diff
@@ -2,33 +2,52 @@
 
 [![GitHub Build Status](https://github.com/cisagov/cool-configure-aws-account/workflows/build/badge.svg)](https://github.com/cisagov/cool-configure-aws-account/actions)
 
-This is a generic skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) [Terraform
-module](https://www.terraform.io/docs/modules/index.html) GitHub
-repository started.  This skeleton project contains [licensing
-information](LICENSE), as well as [pre-commit
-hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
-appropriate for the major languages that we use.
-
-See [here](https://www.terraform.io/docs/modules/index.html) for more
-details on Terraform modules and the standard module structure.
+This repository contains Terraform code for configuring AWS accounts
+for use in the COOL.
+
+## Pre-requisites ##
+
+- A valid AWS profile that has permissions to administer Single Sign-On (SSO)
+  resources, similar to
+  [this policy](https://github.com/cisagov/cool-accounts/blob/develop/master/administersso_policy.tf).
+- [Terraform](https://www.terraform.io/) installed on your system.
+- [jq](https://stedolan.github.io/jq/) installed on your system.
```

Comment:
Due to the `local-exec` calls in this configuration I think we should explicitly call out the AWS CLI as a requirement.

```suggestion
- The [AWS CLI](https://aws.amazon.com/cli/) installed on your system.
- [jq](https://stedolan.github.io/jq/) installed on your system.
```